pkg/netpol/connlist/connlist_test.go

```diff
@@ -1139,4 +1140,229 @@ var goodPathTests = []struct {
 		testDirName:   "netpol_named_port_test_2",
 		outputFormats: []string{output.DefaultFormat},
 	},
+	// tests with adminNetworkPolicy
+	{
+		testDirName:   "anp_test1_deny_traffic_at_cluster_level",
+		outputFormats: []string{output.TextFormat},
+	},
+	{
+		testDirName:   "anp_test2_allow_traffic_at_cluster_level",
```

Comment:
for test `tests/anp_test2_allow_traffic_at_cluster_level`, the result connectivity will be the same even if the policy is deleted, so the effect of the policy is not well demonstrated.
better change the test so that the effect of the tested policy is actual.
please also check this for all other tests.

Reply:
okay, I remember those 3 were the first tests I've added ; were taken from [here](https://network-policy-api.sigs.k8s.io/reference/examples/).
will add `NetworkPolicy` that blocks some of the traffic and see that it is allowed because of the ANP.

will check for other tests

Reply:
done

Reply:
not sure I understand `anp_test_3` - the output with and without the anp is the same

Reply:
true, I jsut wanted to show that if the anp passes a connection but this connection is not allowed by networkpolicy then it is not allowed.
I could add A BANP that denies TCP 8080 ; but we have such example

Reply:
done

Reply:
test  `tests/anp_test_6_swapping_rules`, rule  `"deny-from-slytherin-at-port-5353"`  - better add another port for this rule, so the output shows the effect of both "pass" and "deny" rules instead of juts "pass". (so that only part of the deny rule is overridden and not entirely).

Reply:
done